pkgs/tools/nix/dnadd/default.nix

```diff
@@ -0,0 +1,23 @@
+{stdenv, fetchFromGitHub, gcc}:
+
+stdenv.mkDerivation rec {
+  pname = "dnadd";
+  version = "1.0.0";
+  ownerj="JoeLancaster";
+  src = fetchFromGitHub {
+    owner = ownerj;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ljskfhvc68rw4w5vwknrddknh1xk3iw1z635am7si3x0k9m4ngr";
+  };
+  installPhase = ''
+                 mkdir -p $out
+                 make DESTDIR=$out install
```

Comment:
Also, you can create the directory in `preInstall` and just use the default `installPhase`, passing the flags using `makeFlags` or `installFlags`.

Or better yet, use `install -D` in the `Makefile` so the `preInstall` is not even necessary.